extensions/notification/NotificationQueuePopulator.js

```diff
@@ -128,18 +47,18 @@ class NotificationQueuePopulator extends QueuePopulatorExtension {
      * @param {String} type - entry type
      * @return {undefined}
      */
-    _processObjectEntry(bucket, key, value, type) {
+    async _processObjectEntry(bucket, key, value, type) {
         const versionId = value.versionId || null;
         if (!isMasterKey(key)) {
             return undefined;
         }
-        const config = this.bnConfigManager.getConfig(bucket);
+        const config = await this.bnConfigManager.getConfig(bucket);
         if (config && Object.keys(config).length > 0) {
             const { eventMessageProperty }
                 = notifConstants;
             let eventType
                 = value[eventMessageProperty.eventType];
-            if (eventType === undefined && type === 'del') {
+            if (eventType === undefined && type === 'delete') {
```

Comment:
Where is this change from? (hey, another TS literal type waiting to happen)

Reply:
I'm not actually sure on when or why it got changed, but you can see the part of code that sets the type of the entries  here 

https://github.com/scality/Arsenal/blob/019907e2ab841acc70a32331fec77d99efc4265c/lib/storage/metadata/mongoclient/ListRecordStream.js#L120-L126